src/objects/api/v2/urls.py

```diff
@@ -16,20 +19,23 @@
 app_name = "v2"
 
 urlpatterns = [
-    path("", SpectacularJSONAPIView.as_view(), name="schema-json"),
     path(
         "/",
         include(
             [
-                # schema
                 path(
-                    "schema/openapi.yaml",
```

Comment:
Could you add a redirect from the old path to this new path? Ideally with a deprecationwarning that this will be removed in the next major release